src/ctapipe/image/toymodel.py

```diff
@@ -375,20 +375,88 @@ def pdf(self, x, y):
 class RingGaussian(ImageModel):
     """A shower image consisting of a ring with gaussian radial profile.
 
-    Simplified model for a muon ring.
+    Simplified model for a muon ring. With asymmetry in both the x and y directions.
     """
 
-    def __init__(self, x, y, radius, sigma):
+    def __init__(
+        self,
+        x,
+        y,
+        radius,
+        sigma,
+        asymmetry_magnitude=0,
+        asymmetry_orientation_angle_deg=0 * u.rad,
```

Comment:
`asymmetry_orientation_angle_deg=0 * u.rad,` looks weird semantically. 
```suggestion
        asymmetry_orientation_angle=0 * u.rad,
```